look-and-feel.md

```diff
@@ -0,0 +1,65 @@
+# Look and Feel
+
+### Name and wordmark
+
+Our name uses title case with the following variations:
+
+> **HYPHA**  
+> **HYPHA** Worker Co-operative  
+> Hypha Worker Co-operative Inc. [our legal name]  
+
+We use the following wordmark in online spaces, services and social media where we represent the organization (e.g., [Twitter](https://twitter.com/hyphacoop) and [LinkedIn](https://ca.linkedin.com/company/hyphacoop)). 
```

Comment:
`data-proofer-ignore` attr could go on the LinkedIn anchor to get around the proofer error.

Reply:
gitbook uses a custom parser based on kramdown from what I can tell (most docs disappearing), so I can't just pass this with the {: } syntax. I'm resolving this problem by moving where the content is (not essential here anyway). :wink: